examples/hello_world/run

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-$PWD/scripts/run_example hello_world
+$PWD/scripts/run_example "$@" hello_world
```

Comment:
How about we just get rid of these `run` scripts and just call `run_example` with appropriate parameters?

Reply:
I'm in two minds about that &ndash; it would reduce the number of shell scripts lying around, but it makes each example a little less self-contained.

(I also wonder whether we should separate out the C++ & Rust versions of `hello_world` and then simplify `run_example` to auto-detect the module language and so need no options anyway&hellip;)